shell/models/management.cattle.io.user.js

```diff
@@ -183,19 +180,32 @@ export default class User extends HybridModel {
     return true;
   }
 
+  get asyncNorman() {
+    return this.$dispatch('rancher/find', {
+      type: NORMAN.USER,
+      id:   this.id,
+    }, { root: true });
+  }
+
   get norman() {
-    return this.$rootGetters['rancher/byId'](NORMAN.USER, this.id);
+    return (async() => {
+      const norman = await this.asyncNorman;
+
+      return norman;
+    })();
   }
 
   get canDelete() {
-    return this.norman?.hasLink('remove') && !this.isCurrentUser;
+    return this.hasLink('remove') && !this.isCurrentUser;
```

Comment:
Haven't run through this properly, but as updates to the user go via the norman user (and the steve user permissions are slightly odd) this and canUpdate should continue to be through the norman user